packages/cid-verifier/src/verification.js

```diff
@@ -133,7 +133,8 @@ export const verificationPost = withRequiredQueryParams(['cid', 'url'],
         const threats = evaluateJson?.scores?.filter(score => !env.GOOGLE_EVALUATE_SAFE_CONFIDENCE_LEVELS.includes(score.confidenceLevel)).map(score => score.threatType)
         if (threats.length) {
           await pRetry(
-            () => env.DENYLIST.put(cid, JSON.stringify({
+            async () => env.DENYLIST.put(cid, JSON.stringify({
+              anchor: await toDenyListAnchor(cid),
```

Comment:
```suggestion
            async () => env.DENYLIST.put(toDenyListAnchor(cid), JSON.stringify({
```

Reply:
It should be the key. See https://github.com/web3-storage/reads/blob/7700f4600cc481e9417cfb60a7d98718a9b56319/packages/cid-verifier/src/utils/denylist.js#L31